sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -235,7 +235,7 @@ private SentryTransaction processTransaction(
     for (final EventProcessor processor : eventProcessors) {
       try {
         transaction = processor.process(transaction, hint);
-      } catch (Exception e) {
```

Comment:
Do we have other places in the codebase we should make this change?

Reply:
maybe yes...